### PR TITLE
improve cryptsetup messaging

### PR DESCRIPTION
**Pull request by poettering**
A fix for #11498

### CODE REVIEW

**Comment by poettering** [comment]
Let's postpone this one to v242, since it changes more than just one simple string, and leat's leave 241 for mere bugfixing, in particular as the bug this fixes isn't particularly bad.

File: src/cryptsetup/cryptsetup.c [hunk 1]
```diff
@@ -702,9 +715,7 @@ static int run(int argc, char *argv[]) {
                                 continue;
                         }
                         if (r != -EPERM)
```
Comment (keszybz) [inline src/cryptsetup/cryptsetup.c]: Shouldn't this be `-EAGAIN` now?

**Comment by keszybz** [comment]
FWIW, I wouldn't mind merging this for 241. It seems harmless enough.

File: src/cryptsetup/cryptsetup.c (hunk 1 above)
Comment (poettering) [reply src/cryptsetup/cryptsetup.c]: indeed.

**Comment by poettering** [comment]
well, it does change execution flow, so i was a bit afraid of marking this for 241, given that i didn't test the tcrypt/veracrypt codepaths myself...

Anyway, if you feel comfortable, by all means, merge!

I have update the PR now, with the error code fix.

**Comment by keszybz** [comment]
This looks safe enough. Let's merge it for v241-rc2.

**Pull request merged**